package.json

```diff
@@ -30,6 +30,7 @@
     "next-auth": "^4.12.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.18.0",
```

Comment:
Do we still use `react-router-dom`? If not, I think we should uninstall this package.